### PR TITLE
feat: integrate wolfsburg campus #359

### LINKED ISSUE #359: feat: integrate wolfsburg campus
### Is your feature request related to a problem? Please describe

I want to add the wolfsburg campus to the map . And I cannot commit . As it showed me this:
image

Also the cluster map photos are here.
cluster1
cluster2

### Describe the solution you'd like

I would require your help. As I am doing the commit in github codespaces. 
As for the permission from the staff I can get that for you.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mdabir1203**
**Relative Issues:** Resolve #359

**Describe the pull request**
Added the Wolfsburg campus in the cluster map.

**Checklist**

- [X] I have linked the relative issue to this pull request
- [ ] I have made the modifications or added tests related to my PR
- [ ] I have added/updated the documentation for my RP
- [ ] I put my PR in Ready for Review only when all the checklist is checked

**Breaking changes ?**
yes/no

**Additional context**
Cluster map configuration row might have slight mistakes

### CODE REVIEW

**Comment by mdabir1203** [comment]
Cluster map design is finalized and fixed.

**Comment by 42Atomys** [comment]
@mdabir1203 Thanks for your contribution, I will review your work in the week / by the week end. :)

If you have any staff contact on your campus, tell him to send me a message on 42 Slack to validate with him the integration (legal stuffs)

**Comment by mdabir1203** [comment]
@42Atomys Did anyone reach out to you and also can you provide me with your slack name

**Comment by 42Atomys** [comment]
@mdabir1203 Hello, do you have any news about 42 Wolfsburg ?
I try to go forward with pull requests.

Thanks you in advance !

**Comment by mdabir1203** [comment]
@42Atomys  I will get back to them again.

**Comment by mdabir1203** [comment]
@42Atomys I spoke with them and they said we still need 2/3 weeks max as the linux migration is ongoing which will also change the cluster map

**Comment by 42atomys** [comment]
@mdabir1203 Hi, I've returned to inquire about the status of the "cluster map" change. I've noticed that this pull request has been inactive due to lack of response. Please be advised that without a response, this pull request will automatically close in 30 days.

I will handle all communications and approval processes with the staff members. There's no need for you to make further inquiries.

**Comment by mdabir1203** [comment]
Hey there was no visible change and I believe they were stalling it for no reason :D

**Review by mdabir1203 — COMMENTED** [review]
fixed the merge conflicts

File: web/ui/src/lib/clustersMap/types.d.ts [hunk 1]
```diff
@@ -8,8 +8,9 @@ export type CampusNames =
   | 'vienna'
   | 'seoul'
   | 'lausanne'
-  | 'mulhouse'
-  | 'tokyo';
+  | 'tokyo'
+  | 'wolfsburg'
+  | 'mulhouse';
```
Comment (42atomys) [inline web/ui/src/lib/clustersMap/types.d.ts:13]:
```suggestion
  | 'mulhouse'
  | 'wolfsburg';
```

**Review by 42atomys — APPROVED** [review]

**Comment by mdabir1203** [comment]
Is there anything required to fix that frontend merging fix from my side ?

File: web/ui/src/lib/clustersMap/campus/wolfsburg.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import { Campus } from '../campus';
+import { Cluster } from '../cluster';
+import { CampusNames, ICampus } from '../types';
+
+//
+export class Malaga extends Campus implements ICampus {
```
Comment (42atomys) [inline web/ui/src/lib/clustersMap/campus/wolfsburg.ts:6]: I don't see that, wrong class naming 👀

ℹ️ You can run `yarn make-pretty` to lint your changes

**Review by 42atomys — CHANGES_REQUESTED** [review]

**Comment by 42atomys** [comment]
Really sorry, thanks to unit test to raise the point, the class name has miss named, and I don't see that on the multiple reviews I made..

**Comment by 42atomys** [comment]
@mdabir1203 Great news ! staff approve it 🎉 

I will update the branch and merge it, wolfsburg campus will be present in 0.31 💯 
You can start to spread the love 💜

**Review by 42atomys — APPROVED** [review]

**Pull request merged**